app/views/payment/student-license/PurchaseView.vue

```diff
@@ -233,6 +315,12 @@ export default {
     getSelectedPrice () {
       return this.priceData.find((p) => p.id === this.selectedPrice)
     },
+    async sendVerificationLink () {
+      if (!this.isTecmilenioEmailValid()) {
+        return false
```

Comment:
we use this.errMsg to show the error like in onPurchaseNow otherwise how will user know there is an error 😅